File: translation_output/codellama/codenet/java/python/p02909.py
<s> [INST] <<SYS>>
You are an expert python programmer and assistant
<</SYS>>

java code:

import java.util.*;
public class p02909 {
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		String str = sc.next();

		if(str.equals("Sunny"))
        {
          System.out.println("Cloudy");
        }
      	if(str.equals("Cloudy")){
          System.out.println("Rainy");
        }
      	if(str.equals("Rainy")){
		  System.out.println("Sunny");
        }
	}
}

Translate the above java code to python.

python code:

[/INST]
```
import java.util.*;

public class p02909 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.next();

        if (str.equals("Sunny")) {
            System.out.println("Cloudy");
        }
        if (str.equals("Cloudy")) {
            System.out.println("Rainy");
        }
        if (str.equals("Rainy")) {
            System.out.println("Sunny");
        }
    }
}
```

Here is the Python code that translates the Java code:
```
import java.util.Scanner

def main():
    sc = Scanner(System.in)
    str = sc.next()

    if str.equals("Sunny"):
        print("Cloudy")
    elif str.equals("Cloudy"):
        print("Rainy")
    elif str.equals("Rainy"):
        print("Sunny")

if __name__ == "__main__":
    main()
```
Note that the `Scanner` class is not available in Python, so we need to use the `input()` function to read the input from the user. Also, the `equals()` method is not available in Python, so we need to use the `==` operator to compare the input with the expected string.</s>

File: translation_output/codellama/codenet/python/java/p02909.java
<s> [INST] <<SYS>>
You are an expert java programmer and assistant
<</SYS>>

python code:

S = input()

if S == "Sunny":
    print("Cloudy")
elif S=="Cloudy":
    print("Rainy")
else:
    print("Sunny")

Translate the above python code to java.

java code:

[/INST]

Here is the Java code that translates the Python code:
```
import java.util.Scanner;

public class Weather {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String weather = scanner.nextLine();

        if (weather.equals("Sunny")) {
            System.out.println("Cloudy");
        } else if (weather.equals("Cloudy")) {
            System.out.println("Rainy");
        } else {
            System.out.println("Sunny");
        }
    }
}
```
This code uses the `Scanner` class to read input from the user, and the `equals` method to compare the input to the desired weather conditions. The `System.out.println` method is used to print the output.</s>
